### PR TITLE
Schedule: Tests 

### LINKED ISSUE #191: Create tests for schedule

### PR DESCRIPTION
**Pull request by Guofeng-Tang**
resolve #191

### CODE REVIEW

File: src/main/java/seedu/zerotoone/model/schedule/RecurringSchedule.java [hunk 1]
```diff
@@ -1,58 +1,58 @@
...
+//         if (!(other instanceof RecurringSchedule)) {
+//             return false;
+//         }
+//
+//         RecurringSchedule otherSchedule = (RecurringSchedule) other;
+//         return otherSchedule.workoutToSchedule.equals(workoutToSchedule)
+//                 && otherSchedule.startDate.equals(startDate)
+//                 && otherSchedule.endDate.equals(endDate);
+//     }
+// }
```
Comment (alloystory) [inline src/main/java/seedu/zerotoone/model/schedule/RecurringSchedule.java]: Why did you comment out this file? Please remove this file, and if you want to recreate this class, then bring it back from version history.

**Review by alloystory — CHANGES_REQUESTED** [review]
Noted changes. Please make the edit as mentioned.

File: src/main/java/seedu/zerotoone/model/schedule/RecurringSchedule.java (hunk 1 above)
Comment (Guofeng-Tang) [reply src/main/java/seedu/zerotoone/model/schedule/RecurringSchedule.java]: ok will do

**Review by alloystory — APPROVED** [review]
boleh!

**Pull request merged**